client/asset/eth/node.go

```diff
@@ -192,12 +191,18 @@ func prepareNode(cfg *nodeConfig) (*node.Node, error) {
 	return node, nil
 }
 
-func ethChainConfig(network dex.Network) (c ethconfig.Config, err error) {
+// TODO: Consider polygon support on mainnet and testnet.
+func chainConfig(chainID int64, network dex.Network) (c ethconfig.Config, err error) {
```

Comment:
@buck54321, I'm thinking these are the correct genesis configs for mainnet and testnet respectively, right? https://github.com/maticnetwork/bor/blob/891ec7fef619cac0a796e3e29d5b2d4c095bdd9b/internal/cli/server/chains/mainnet.go#L11-L13, https://github.com/maticnetwork/bor/blob/891ec7fef619cac0a796e3e29d5b2d4c095bdd9b/internal/cli/server/chains/mumbai.go#L11

Reply:
There's also https://github.com/maticnetwork/bor/tree/891ec7fef619cac0a796e3e29d5b2d4c095bdd9b/core/allocs. Not sure the difference.

Reply:
> There's also https://github.com/maticnetwork/bor/tree/891ec7fef619cac0a796e3e29d5b2d4c095bdd9b/core/allocs. Not sure the difference.

This one contains only allocation config. From our use of `eth`  mainnet and testnet genesis config, I think we'd need the whole genesis cfg in the links cited in my comment.

Reply:
/internal/cli/server/chain seems like such an odd place to store genesis files, but whatever.

Reply:
> /internal/cli/server/chain seems like such an odd place to store genesis files,

yh, that was why I asked to be sure of what we'd use :)

Reply:
Also seen: https://github.com/maticnetwork/bor/blob/develop/core/genesis.go#L447

Reply:
> @buck54321, I'm thinking these are the correct genesis configs for mainnet and testnet respectively, right? https://github.com/maticnetwork/bor/blob/891ec7fef619cac0a796e3e29d5b2d4c095bdd9b/internal/cli/server/chains/mainnet.go#L11-L13, https://github.com/maticnetwork/bor/blob/891ec7fef619cac0a796e3e29d5b2d4c095bdd9b/internal/cli/server/chains/mumbai.go#L11

It seems these are just a copy of the Genesis configs, plus some optional chain cfg.

Reply:
@buck54321, I tried to import `bor` but they didn't change their module part after forking Ethereum.  Do you have any workaround in mind?

```
go get github.com/maticnetwork/bor@v0.2.18-0.20220922050621-c91d4ca1fa4f

go: downloading github.com/maticnetwork/bor v0.2.18-0.20220922050621-c91d4ca1fa4f
go: github.com/maticnetwork/bor@v0.2.18-0.20220922050621-c91d4ca1fa4f: parsing go.mod:
        module declares its path as: github.com/ethereum/go-ethereum
                but was required as: github.com/maticnetwork/bor
                
  ```

Reply:
> EDIT: We might end up copying the genesis configs. Is this okay?

If that's all we're trying to import `github.com/maticnetwork/bor` for, then yes, just copy.
They clearly don't intend people to be importing it or they would have made a usable module.

Reply:
> If that's all we're trying to import github.com/maticnetwork/bor for, then yes, just copy.

yeah.